internal/backends/mcu/pico_st7789.rs

```diff
@@ -31,6 +31,9 @@ static mut HEAP: [u8; HEAP_SIZE] = [0; HEAP_SIZE];
 #[global_allocator]
 static ALLOCATOR: CortexMHeap = CortexMHeap::empty();
 
+// 16ns for serial clock cycle (write), page 43 of https://www.waveshare.com/w/upload/a/ae/ST7789_Datasheet.pdf
+const SPI_ST7789VW_MAX_FREQ: u32 = 62_500_000u32;
```

Comment:
Too bad this is not in the Hertz type already

```suggestion
const SPI_ST7789VW_MAX_FREQ: embedded_time::Hertz = embedded_time::Hertz(62_500_000);
```

Reply:
Ah yes, good idea. The above doesn't quite compile, but I've pushed the equivalent as follow-up.